### DISCUSSION TITLE
Possible to hide this downloading notification

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [deniscerri/ytdlnis#319] [General]
![This Downloading one? I turned off downloading notification from settings already but it didn't remove it](https://github.com/deniscerri/ytdlnis/assets/38470944/a8fde44d-d286-48ee-abd8-530906f3e57c)

I am on android 12l
Redmi note 10s

**Comment by deniscerri** [comment]
its necessary for the download not to be destroyed in the background. Its a foreground notification telling android its important.

**Reply by HikaruIchijyo** [reply]
Oh ok got it
